### PR TITLE
Bs2 40 fe connect sign up aws

### PR DESCRIPTION
**Pull request by tapabratadey**

### CODE REVIEW

File: src/pages/api/user.ts [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
+      // go to the get routes file import it from routes/get.ts
+      //Basic ping/pong endpoint to test server functionality
+      user_controller(req, res, user);
+    } else {
+      console.log(req.body);
+      // Handle any other HTTP method
+    }
+  } catch (error) {
+    console.log(error);
+    res.status(401).json({ error: error });
```
Comment (bennettl) [inline src/pages/api/user.ts:27]: .json({ error });

File: src/pages/app/profile/index.tsx [hunk 2]
```diff
@@ -14,14 +34,42 @@ const MyProfile: NextPage = () => {
         <link rel="icon" href="/favicon.ico" />
       </Head>
 
-      <Navbar />
-
       <Box className={styles.signup_wrapper}>
         <h1>MyProfile</h1>
+        {authenticated && (
```
Comment (bennettl) [inline src/pages/app/profile/index.tsx:39]: Wouldn't the user always be authenticated?

File: src/pages/signup.tsx [hunk 3]
```diff
@@ -29,48 +32,29 @@ const Signup: NextPage = () => {
...
-    } catch (err) {
-      console.log(err);
-      console.log(err);
+        attributes: {
+          given_name: fName,
+          family_name: lName,
+          email,
+        },
+      });
+      console.log(user);
```
Comment (bennettl) [inline src/pages/signup.tsx:54]: Let's clear up console.logs to not pollute the console unless it's required (e.g. error logging)

File: src/pages/verify.tsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    } catch (error) {
+      console.log('error signing in', error);
+    }
+    setIsLoggingIn(false);
+  };
+
+  return (
+    <>
+      <Head>
+        <title>JumpStarter - Login</title>
```
Comment (bennettl) [inline src/pages/verify.tsx:45]: Should not have head tag

File: src/pages/verify.tsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+              type="submit"
+              onClick={handleLogin}
+              className={styles.login_button}
+            >
+              Confirm
+            </button>
+          </div>
+        </div>
+        <div className={styles.account_not_exists}>
+          Don&apos;t have an account?
```
Comment (bennettl) [inline src/pages/verify.tsx:90]: `Don&apos;t`

File: src/pages/app/profile/index.tsx (hunk 2 above)
Comment (tapabratadey) [reply src/pages/app/profile/index.tsx:39]: Yep! Had that before adding the redirect link, removed it now. Thanks!

File: src/pages/api/user.ts (hunk 1 above)
Comment (tapabratadey) [reply src/pages/api/user.ts:27]: Thanks, made changes.

File: src/pages/verify.tsx (hunk 4 above)
Comment (tapabratadey) [reply src/pages/verify.tsx:45]: Fixed. Merged @marri-kang's changes.

File: src/pages/verify.tsx (hunk 5 above)
Comment (tapabratadey) [reply src/pages/verify.tsx:90]: It was an eslint error. Fixed, added /* eslint-disable react/no-unescaped-entities */ to file.

File: src/pages/verify.tsx [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+        <title>JumpStarter - Login</title>
+        <meta name="description" content="Lets JumpStart projects" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <Box className={styles.login_wrapper}>
+        <h1>Confirm Sign Up</h1>
+        <div>
+          <input
+            type="text"
```
[the remark this answers is not in the document]
Comment (tapabratadey) [reply src/pages/verify.tsx:53]: Fixed. Replaced HTML <div> <h1> <p> <button> <input> with Grommet tags.

**Comment by tapabratadey** [comment]
Thanks a lot @bennettl for your detailed feedback and patience, really appreciate it! Please let me know if these changes look okay.

**Review by bennettl — APPROVED** [review]

**Pull request merged**